users/spidey3/spidey3.h

```diff
@@ -9,12 +9,14 @@
 enum userspace_layers {
   _BASE = 0,
   _OSX,
+  _NUMPAD,
   _FN,
 };
 
 enum rgb_base_layer {
   RGB_LAYER_BASE_DEFAULT = _BASE,
-  RGB_LAYER_BASE_REGULAR = _FN,
+  RGB_LAYER_BASE_REGULAR = _NUMPAD,
+  RGB_LAYER_BASE_ACKS = _FN+1,
```

Comment:
Given `userspace_layers` above, do you have a layers above `_FN`?

Reply:
Nothing above _FN right now.
This is really just about mapping the keymap layers to rgblight layers.

Reply:
Ah - if you are confused about the _FN+1, that's because the yes/no ACK blinks use rgblight layers that are above the _FN rgblight layer.

Reply:
I wouldnt personally mix this enum with the layers enum and offsets, unless it was a 1:1 mapping ( I would say the 2 things dont really key together nicely in this scenario). But then its not my keymap....

Reply:
Yeah - I was thinking the same thing; that's resolved somewhat in the _next_ PR, that does some further cleanup, and also adds a cool way to do timed "blinks" of a rgblight layer.